pkg/daemon/controlplane.go

```diff
@@ -0,0 +1,79 @@
+package daemon
+
+// This file provides changes that we make to the control plane
+// only.
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+// setRootDeviceSchedulerBFQ switches to the `bfq` I/O scheduler
+// for the root block device to better share I/O between etcd
+// and other processes.  See
+// https://github.com/openshift/machine-config-operator/issues/1897
+// Note this is the current systemd default in Fedora, but not RHEL8,
+// except for NVMe devices.
```

Comment:
So why wouldn't we do this for all root devices on all nodes in the cluster?

Reply:
Mostly to limit the "blast radius" of this PR - changing the control plane only minimizes risk.  But, it does also mean more divergence.

It probably does make sense to do an across-the-board switch though...maybe as a followup to this?  Or would you rather do it in one go?

Reply:
The other aspect is that currently the OS update vs etcd happens because we're updating the OS while etcd is still running.  But for regular worker nodes that don't have static pods, everything will have been drained before we start the update - so there's less need for bfq inherently.

(Long term I think we really want cgroups v2 + a filesystem that honors IO priorities - it's the only way to get real balance, bfq is mostly heuristics + support for baseline io priorities)

Reply:
@cgwalters @smarterclayton I would suggest to sync up with the perf-dept, there were some concerns regarding BFQ as the default scheduler in RHEL. There are several workloads where BFQ did not perform the best. MQ-deadline did better. There are probably more workloads that need to be strobed. 

Even though BFQ is great on the low end, past experience shows it doesn't do as well on the high end.  It would need to be thoroughly tested if RHCOS considered changing it.  See http://post-office.corp.redhat.com/archives/kscale-list/2019-September/msg00010.html

Reply:
The BFQ may work great for the control-plane where we're only running "one" workload (etcd), on the worker nodes we have a variety of workloads that may or not benefit from BFQ. Would also be interesting to see how the infra pods are reacting to such a change.

Reply:
Thank you for raising the concern. This PR uses BFQ only for control plane nodes. We will keep in mind and make sure to talk to perf team if we plan to use BFQ for worker/custom pools.